ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml

```diff
@@ -149,7 +150,11 @@ StatusDialog {
             }
         }
         leftButtons: ObjectModel {
-            StatusBackButton { id: backButton }
+            StatusBackButton {
```

Comment:
Is it just this problematic popup, or do we have a similar problem somewhere else too? (we do have a back button in other places too)

Reply:
Cool! Pls also make sure to squash the 2 commits